app/util/csrf.test.ts

```diff
@@ -9,7 +10,7 @@ import { createSession } from "@remix-run/node";
 describe("appendCsrfToken", () => {
   describe("with empty session", () => {
     test("token is added", () => {
-      const session = createSession({});
+      const session: Session = createSession({});
```

Comment:
Why could the type not be applied by the return type of createSession? Same for all the other calls to createSession in this file?

Reply:
that's a good question. maybe could you help me?

Reply:
Yes, let's maybe talk about it tomorrow? I was just wondering whether the tests failed for you before you added this because on my machine the tests were running in both cases

Reply:
sure. I just re-ran again freshly, I do not get the error anymore, but my editor shows the error (they were error too) 
![image](https://user-images.githubusercontent.com/9930966/229543736-6087e3d2-a6ef-4f4b-a7fa-a10ee75c264d.png)

Reply:
ok. I got the error again, let's have a chat tomorrow!